src/components/molecules/OakCollapsibleContent/__snapshots__/OakCollapsibleContent.test.tsx.snap

```diff
@@ -22,6 +22,7 @@ exports[`OakCollapsibleContent matches snapshot 1`] = `
   max-height: 100%;
   padding-right: 1rem;
   font-family: --var(google-font),Lexend,sans-serif;
+  overflow: auto;
```

Comment:
Some of these snapshots automatically updated when I added overflow to the typography styles